templates/vue/src/components/modals/NodeModal/forms/ContentForm/MultipleChoiceForm.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+  <div>
+    <b-form-group :label="multipleAnswerSelected ? 'Checkbox Form' : 'Radio Form'">
```

Comment:
Is the label really needed here?, both forms are the same. If so I think we should change it to "Multiple Answers Form" and "Single Answer Form" as not all users know what radio buttons are